services/reporting/service.go

```diff
@@ -39,9 +39,9 @@ type Service struct {
 	wg            sync.WaitGroup
 }
 
-func NewService(c Config, token string, l *log.Logger) *Service {
-	client := client.New(token)
-	client.URL = c.EnterpriseURL
+func NewService(c Config, l *log.Logger) *Service {
+	client := client.New("")
```

Comment:
@nathanielc Yep, that's fine :scissors: . The empty string token is the same approach we've taken in Chronograf and Core for usage-only reporting.